hpobench/benchmarks/ml_mmfb/tabular_benchmark.py

```diff
@@ -0,0 +1,230 @@
+
+    # pylint: disable=arguments-differ
+    @AbstractBenchmark.check_parameters
+    def objective_function(self,
+                           configuration: Union[CS.Configuration, Dict],
+                           fidelity: Union[Dict, CS.Configuration, None] = None,
+                           rng: Union[np.random.RandomState, int, None] = None,
+                           seed: Union[int, None] = None,
+                           metric: Union[str, None] = 'acc',
+                           **kwargs) -> Dict:
+
+        result = self._objective(configuration, fidelity, seed, metric, evaluation="val")
+        return result
+
+    # pylint: disable=arguments-differ
+    @AbstractBenchmark.check_parameters
+    def objective_function_test(self,
+                                configuration: Union[CS.Configuration, Dict],
+                                fidelity: Union[Dict, CS.Configuration, None] = None,
+                                rng: Union[np.random.RandomState, int, None] = None,
+                                seed: Union[int, None] = None,
+                                metric: Union[str, None] = 'acc',
+                                **kwargs) -> Dict:
+
+        result = self._objective(configuration, fidelity, seed, metric, evaluation="test")
+        return result
+
+    # pylint: disable=arguments-differ
+    def get_configuration_space(self, seed: Union[int, None] = None) -> CS.ConfigurationSpace:
+        raise NotImplementedError
+
+    # pylint: disable=arguments-differ
+    def get_fidelity_space(self, seed: Union[int, None] = None) -> CS.ConfigurationSpace:
+        raise NotImplementedError
+
+    # pylint: disable=arguments-differ
+    def get_meta_information(self) -> Dict:
+        """ Returns the meta information for the benchmark """
+        return {'name': 'BaseTabularBenchmark',
+                'references': [],
+                'task_id': self.task_id,
+                'model': self.model
+                }
+
+    def _preprocess_configspace(self, config_space: CS.ConfigurationSpace) -> CS.ConfigurationSpace:
+        """ Converts floats to np.float32 """
+        for hp in config_space.get_hyperparameters():
+            hp.sequence = tuple(np.array(hp.sequence).astype(np.float32))
+            hp.default_value = np.float32(hp.default_value)
+        return config_space
+
+    def _total_number_of_configurations(self, space: str = "hyperparameters") -> int:
+        """ Returns the number of unique configurations in the parameter/fidelity space
+        """
+        count = 1
+        cs = self.configuration_space if space == "hyperparameters" else self.fidelity_space
+        for hp in cs.get_hyperparameters():
+            count *= len(hp.sequence)
+        return count
+
+    def _seeds_used(self) -> List:
+        return self.table.seed.unique().tolist()
+
+    def sample_hyperparamer(self, n: int = 1) -> Union[CS.Configuration, List]:
```

Comment:
a typo from my end, could you please fix this 
should ofc be `sample_hyperparameter`